lib/generators/rspec/scaffold/scaffold_generator.rb

```diff
@@ -60,8 +78,6 @@ def generate_routing_spec
         template 'routing_spec.rb', template_file
       end
 
-      hook_for :integration_tool, as: :integration
```

Comment:
(Check also my commit message on this change)

I had to dig extremely dig into the git history of the project, tried to figure the purpose and the consequences of this setting (the generator from `railties` does not have this). But I don't understand it. But is the thing that causes the `IntegrationGenerator` to run and also generate a `request_spec` file (figured that out through debugging and lookig at the backtrace).

Changing this seems to cause *something*, at least that's what the test suite is telling me

See also 98b886b5528aa2a60c42d80b83a0973f24bfcdb7